docs/MapColonies/auth.md

```diff
@@ -13,17 +13,36 @@ tags:
 
 # Authentication
 
-In order to make requests either as a service or a client, you will need to identify yourself with a token.
-The token, should be used in **every** request to our services and will enable you to access our services in accordance to the permissions that were given to you. <br/>
+In order to make requests either as a service or a client, you will need to identify yourself with a `token`.
+The `token` should be used in **every** request to our services and will allow you to access our services in accordance to the permissions that were given to you.<br/>
+
+**All services** will require an `API Token`, it will be specified in each service description.
 
 :::caution
 The token might have an **expiration date**
 :::
 
-When making any request, you need to add a special **header** or **query parameter** in which you supply the token.
-
 :::info
 **Important:** only **query parameter** `token` is currently supported
 :::
 
-**All services** will require an `API Token`, it will be specified in the service description.
+## Supplying the token
+
+When making any request, you need to add a special **header** or **query parameter** in which you supply the token.
+
+| **How we supply the token** | **Name** |
+| ----------- | ----------- |
+| Query | token |
+| Header | x-api-key |
+
+## Using the token as a query parameter
+
+```
+<SERVICE_URL>/SUB/PATH?token=<token>
```

Comment:
Specify that it is a GET request if you don't want to use curl here

Reply:
It doesn't really matter if it is a `GET` or `POST` request, what matters is the query parameter. That's why I think it's irrelevant for this example.

Reply:
I used `curl` for the other example because it's easier to show how to add the `header` that way.